bin/mock-outcomes

```diff
@@ -0,0 +1,68 @@
+#!/usr/bin/env python
+from sentry.runner import configure
+
+configure()
+
+from datetime import datetime, timedelta
+
+import requests
+from django.conf import settings
+
+from sentry.constants import DataCategory
+from sentry.utils import json
+from sentry.utils.outcomes import Outcome
+
+# import random
+
+
+def store_outcomes(outcome, num_times=1):
+    outcomes = []
+    for _ in range(num_times):
+        outcome_copy = outcome.copy()
+        outcome_copy["timestamp"] = outcome_copy["timestamp"].strftime("%Y-%m-%dT%H:%M:%S.%fZ")
+        outcomes.append(outcome_copy)
+    assert (
+        requests.post(
+            settings.SENTRY_SNUBA + "/tests/outcomes/insert", data=json.dumps(outcomes)
+        ).status_code
+        == 200
+    )
```

Comment:
This code is more useful:
```diff
diff --git a/bin/mock-outcomes b/bin/mock-outcomes
index 4ac11afcf4..8135e00ba9 100755
--- a/bin/mock-outcomes
+++ b/bin/mock-outcomes
@@ -21,12 +21,9 @@ def store_outcomes(outcome, num_times=1):
         outcome_copy = outcome.copy()
         outcome_copy["timestamp"] = outcome_copy["timestamp"].strftime("%Y-%m-%dT%H:%M:%S.%fZ")
         outcomes.append(outcome_copy)
-    assert (
-        requests.post(
-            settings.SENTRY_SNUBA + "/tests/outcomes/insert", data=json.dumps(outcomes)
-        ).status_code
-        == 200
-    )
+
+    req = requests.post(settings.SENTRY_SNUBA + "/tests/outcomes/insert", data=json.dumps(outcomes))
+    req.raise_for_status()


 def generate_outcomes():
```

After:
```
./bin/mock-outcomes
13:32:03 [WARNING] sentry.utils.geo: settings.GEOIP_PATH_MMDB not configured.
13:32:05 [INFO] sentry.plugins.github: apps-not-configured
0
Traceback (most recent call last):
  File "./bin/mock-outcomes", line 68, in <module>
    generate_outcomes()
  File "./bin/mock-outcomes", line 49, in generate_outcomes
    store_outcomes(
  File "./bin/mock-outcomes", line 24, in store_outcomes
    assert (
AssertionError
```

Before:
```
./bin/mock-outcomes
13:33:44 [WARNING] sentry.utils.geo: settings.GEOIP_PATH_MMDB not configured.
13:33:45 [INFO] sentry.plugins.github: apps-not-configured
0
Traceback (most recent call last):
  File "./bin/mock-outcomes", line 65, in <module>
    generate_outcomes()
  File "./bin/mock-outcomes", line 46, in generate_outcomes
    store_outcomes(
  File "./bin/mock-outcomes", line 26, in store_outcomes
    req.raise_for_status()
  File "/Users/armenzg/code/sentry/.venv/lib/python3.8/site-packages/requests/models.py", line 943, in raise_for_status
    raise HTTPError(http_error_msg, response=self)
requests.exceptions.HTTPError: 405 Client Error: METHOD NOT ALLOWED for url: http://127.0.0.1:1218/tests/outcomes/insert
```

Reply:
What do I need to do to make it work? Do I need to start up some extra dev services?